docs/docs/support/advisory/a10004.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Technical Advisory 10004
+---
+
+## Date and Version
+
+Version: 2.39.0
+
+Date: 2023-10-14
+
+## Description
+
+Due to storage optimisations ZITADEL changes the behaviour of sequences. 
+This change improves command (create, update, delete) performance of ZITADEL.
+
+Sequences are no longer unique inside an instance.
+From now on sequences are upcounting per aggregate id. 
+For example sequences of newly created users begin at 1.
+Existing sequences remain untouched.
+
+## Statement
+
+This change is tracked in the following PR: [new eventstore framework](https://github.com/zitadel/zitadel/issues/5358).
+As soon as the release version is published, we will include the version here.
+
+## Mitigation
+
+If you use the ListEvents API to scrape events use the creation date instead of the sequence.
+
+## Impact
+
+Once this update has been released and deployed, sequences are no longer unique inside an instance.
+ZITADEL will increate parallel write capabilities, because there is no global sequence to track anymore.
```

Comment:
```suggestion
ZITADEL will increate parallel write capabilities, because there is no global sequence to track anymore.
```
```suggestion
ZITADEL will increate parallel write capabilities, because there is no global sequence to track anymore.
```